packages/neos-ui-guest-frame/src/initializePropertyDomNode.js

```diff
@@ -0,0 +1,61 @@
+import {$get} from 'plow-js';
+
+import {actions} from '@neos-project/neos-ui-redux-store';
+
+import {getGuestFrameWindow, closestContextPathInGuestFrame} from './dom';
+
+export default ({store, globalRegistry, nodeTypesRegistry, inlineEditorRegistry, nodes}) => propertyDomNode => {
+    const guestFrameWindow = getGuestFrameWindow();
+    if (!guestFrameWindow['@Neos.Neos.Ui:InitializedInlineEditors']) {
+        guestFrameWindow['@Neos.Neos.Ui:InitializedInlineEditors'] = {};
+    }
+
+    const initializedInlindeEditorApis = guestFrameWindow['@Neos.Neos.Ui:InitializedInlineEditors'];
+    const propertyName = propertyDomNode.getAttribute('data-__neos-property');
+    const contextPath = closestContextPathInGuestFrame(propertyDomNode);
+    const nodeTypeName = $get([contextPath, 'nodeType'], nodes);
+    const nodeType = nodeTypesRegistry.get(nodeTypeName);
+    const isInlineEditable = $get(['properties', propertyName, 'ui', 'inlineEditable'], nodeType) !== false;
+
+    if (isInlineEditable) {
+        const editorIdentifier = nodeTypesRegistry.getInlineEditorForProperty(nodeTypeName, propertyName);
```

Comment:
the method name `getInlineEditorForProperty` doesn't make clear that an "editor identifier" is returned. Suggestion: `getInlineEditorIdentifierForProperty`